hack/jenkins/e2e.sh

```diff
@@ -326,6 +326,13 @@ if [[ "${E2E_UP,,}" == "true" || "${JENKINS_FORCE_GET_TARS:-}" =~ ^[yY]$ ]]; the
             bucket="${varr[0]}"
             githash="${varr[1]}"
             echo "$bucket / $githash"
+        elif [[ ! -z ${JENKINS_USE_SERVER_VERSION:-} ]]; then
```

Comment:
Ah no that's fine, I was just curious whether it was the same name (I think it's good that it is).

Though actually if I could ask you to change one thing: could you make it similar to other logic whether we check for something that looks like `y`? Otherwise if I put

```bash
export JENKINS_USE_SERVER_VERSION=n
```

it will still get run, which is confusing.